NuGet.config

```diff
@@ -11,5 +11,21 @@
     <add key="AzureFunctionsPreRelease" value="https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsPreRelease/nuget/v3/index.json" />
     <!-- dotnet-tools is required for Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit -->
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json"/>
+
+	<!-- For Release NuGet Feed
+	<add key="Slsec" value="https://msazure.pkgs.visualstudio.com/f3247cef-e371-4261-b105-33f9613c6d8e/_packaging/ServerlessSecurity/nuget/v3/index.json" />
+    -->
+	<!-- For Local Build NuGet Feed
+    <add key="SlSecHost" value="C:\repos\SHA-ServerlessSecurity\Platform\src\ServerlessSecurityHost\bin\Debug" />
```

Comment:
Out of curiosity, how is the windows solution going to pull the SLSec nugets from if we're not defining it in the Nuget.Config?

Reply:
Ideally, we will be uploading NuGet to their feed. For our builds created different branches.